src/components/ClassInfo.vue

```diff
@@ -263,7 +263,16 @@ export default {
     subjectsWithPrereq: function () {
       const currentID = this.currentSubject.subject_id;
       const currentDept = currentID.substring(0, currentID.indexOf('.'));
-      var IDMatcher = new RegExp('(^|[^\\da-zA-Z])' + currentID.replace(/[.*+?^${}()|[\]\\]/g, '\\$&') + '(?![\\da-zA-Z])');
+
+      var thisGIRAttr = this.currentSubject.gir_attribute;
+
+      if(thisGIRAttr == undefined){
+        var IDMatcher = new RegExp('(^|[^\\da-zA-Z])' + currentID.replace(/[.*+?^${}()|[\]\\]/g, '\\$&') + '(?![\\da-zA-Z])');
```

Comment:
The IDMatcher variable is declared separately within the if and else blocks.  While this does work fine (Javascript will "hoist" the variable so it exists outside of the if statement), it is better practice/clearer to define this variable above the if statement (just "var IDMatcher;") and then assign to it within the if statement.